docs/pages/guides/tree-shaking.mdx

```diff
@@ -298,7 +300,7 @@ config.transformer.getTransformOptions = async () => ({
 module.exports = config;
 ```
 
-Experimental import support uses Metro's version of the `@babel/plugin-transform-modules-commonjs` plugin. This drastically reduces the number of resolutions and simplifies your output bundle. This feature can be used with `inlineRequires` to further optimize your bundle experimentally.
+Experimental import support uses custom version of the `@babel/plugin-transform-modules-commonjs` plugin. This drastically reduces the number of resolutions and simplifies your output bundle. This feature can be used with `inlineRequires` to further optimize your bundle experimentally.
```

Comment:
```suggestion
Experimental import support uses a custom version of the `@babel/plugin-transform-modules-commonjs` plugin. This drastically reduces the number of resolutions and simplifies your output bundle. This feature can be used with `inlineRequires` to further optimize your bundle experimentally.
```

Side-note, @EvanBacon, is the `inlineRequires` comment still relevant here? I think we could streamline this to explain this in plain terms. This section doesn't feel like it adds much of an explanation relevant to users